pydocx/export/base.py

```diff
@@ -779,7 +826,7 @@ def paragraph(self, text):
         return text
 
     @abstractmethod
-    def heading(self, text, heading_level):
+    def heading(self, text, heading_style_name):
```

Comment:
Do you think we should do that even though we haven't released a stable public API? We're not 1.0.0 yet.